### ISSUE TITLE
The selector "#groupPrice *" on hx-include returned no matches!

### ISSUE THREAD

**Issue by john-henry** [putyourlightson/craft-sprig#202]
### Describe the bug
I just updated to latest Sprig and Craft and on certain sprig includes I am getting this console error

> htmx.min.js:1 The selector "#groupPrice *" on hx-include returned no matches!

My component incldue looks like this

```
{{ sprig('_components/product/handpainted/price', {
	activeProductVariant:activeProductVariant.id,
	dimension:'piece'
}, {id: 'groupPrice','s-trigger': 'load'}) }}
```

I'm guessing contents on include wouldn't affect things

I can DM you a link to live page if needed

### Versions
- Plugin version:1.12.0
- Craft version:3.7.36

**Comment by bencroker** [comment]
This is likely htmx complaining that it doesn't know which element to use if you have multiple Sprig components using the same ID `groupPrice`.

**Comment by john-henry** [comment]
OK will add some uniqueness to them and report back if all ok

**Comment by john-henry** [comment]
Same issue happens when I add a unique ID
![Image 2022-03-04 at 2 09 49 PM](https://user-images.githubusercontent.com/44709/156778411-9d2abfd0-9d49-4747-b33a-3f8f6a50b1f1.png)

**Comment by john-henry** [comment]
If I remove my ID it adds its own and same issue occurs
![Image 2022-03-04 at 2 19 18 PM](https://user-images.githubusercontent.com/44709/156779933-7c1aa6a7-0dc1-493d-8e3f-7d50c5251b5e.png)

**Comment by bencroker** [comment]
I can't replicate this locally. Please test with a single component and feel free to share the simplest code sample that fails here so I can take a closer look.

**Comment by studiobruikbaar** [comment]
@john-henry I got the same error messages and found out that this happens when the included components are wrapped with `{% if sprig.isRequest %}` but don't have a fallback (`{% else %}`). 

The following example works fine and doesn't throw the error:

```
{% if sprig.isRequest %}
    [ ... ]
{% else %}
    <div class="hidden">Content displayed before the trigger.</div>
{% endif %}
```

**Comment by goraxan** [comment]
Same happens to me, but the error stays even after putting the else.

**Comment by studiobruikbaar** [comment]
@goraxan Did you add some content to the `{% else %}` state? That was needed in my case to get rid of the error.

**Comment by goraxan** [comment]
@studiobruikbaar I tried adding text (no html) in the else, adding a `<div></div>` in the else fixes it, thanks!

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-sprig-core/commit/49bf3a0e0e706ab60017e7445b06f3d60837f834 and released in version 1.12.1.
